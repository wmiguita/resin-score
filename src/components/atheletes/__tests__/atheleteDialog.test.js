import React from "react"
import renderer from "react-test-renderer"

import AtheleteDialog from "../atheleteDialog"

describe( "AtheleteDialog", () => {
  it( "should render AtheleteDialog", () => {
    const tree = renderer.create( <AtheleteDialog open={ false }/> ).toJSON()

    expect( tree ).toMatchSnapshot()
  })
})
