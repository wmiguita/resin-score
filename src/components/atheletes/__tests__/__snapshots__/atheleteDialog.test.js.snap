// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtheleteDialog should render AtheleteDialog 1`] = `null`;
